[2025-08-22T18:38:37.600+0000] {processor.py:186} INFO - Started process (PID=59) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:38:37.601+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:38:37.605+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:38:37.604+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:38:38.828+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:38:38.827+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:38:38.829+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:38:38.847+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 1.257 seconds
[2025-08-22T18:39:09.811+0000] {processor.py:186} INFO - Started process (PID=72) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:39:09.812+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:39:09.814+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:39:09.814+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:39:10.467+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:39:10.466+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:39:10.468+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:39:10.484+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.680 seconds
[2025-08-22T18:39:41.017+0000] {processor.py:186} INFO - Started process (PID=85) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:39:41.018+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:39:41.021+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:39:41.021+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:39:41.672+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:39:41.671+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:39:41.673+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:39:41.688+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.677 seconds
[2025-08-22T18:40:12.220+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:40:12.221+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:40:12.224+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:40:12.223+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:40:12.874+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:40:12.873+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:40:12.875+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:40:12.895+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.682 seconds
[2025-08-22T18:40:43.423+0000] {processor.py:186} INFO - Started process (PID=117) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:40:43.470+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:40:43.472+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:40:43.472+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:40:44.111+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:40:44.110+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:40:44.112+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:40:44.127+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.711 seconds
[2025-08-22T18:41:14.626+0000] {processor.py:186} INFO - Started process (PID=130) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:41:14.627+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:41:14.629+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:41:14.629+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:41:15.272+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:41:15.271+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:41:15.273+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:41:15.290+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.671 seconds
[2025-08-22T18:41:46.068+0000] {processor.py:186} INFO - Started process (PID=143) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:41:46.070+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:41:46.074+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:41:46.073+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:41:46.615+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:41:46.613+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:41:46.616+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:41:46.640+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.580 seconds
[2025-08-22T18:42:17.298+0000] {processor.py:186} INFO - Started process (PID=156) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:42:17.300+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:42:17.302+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:42:17.302+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:42:17.784+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:42:17.782+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:42:17.785+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:42:17.805+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-22T18:42:48.576+0000] {processor.py:186} INFO - Started process (PID=169) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:42:48.578+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:42:48.582+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:42:48.581+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:42:49.079+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:42:49.078+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:42:49.080+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:42:49.100+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.534 seconds
[2025-08-22T18:43:19.776+0000] {processor.py:186} INFO - Started process (PID=182) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:43:19.778+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:43:19.782+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:43:19.781+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:43:20.508+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:43:20.506+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:43:20.509+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:43:20.530+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.764 seconds
[2025-08-22T18:43:50.995+0000] {processor.py:186} INFO - Started process (PID=195) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:43:50.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:43:50.999+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:43:50.999+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:43:51.482+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:43:51.480+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:43:51.482+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:43:51.503+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.516 seconds
[2025-08-22T18:44:22.210+0000] {processor.py:186} INFO - Started process (PID=208) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:44:22.212+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:44:22.215+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:44:22.215+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:44:22.704+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:44:22.702+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:44:22.705+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:44:22.724+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-22T18:44:53.424+0000] {processor.py:186} INFO - Started process (PID=221) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:44:53.425+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:44:53.428+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:44:53.428+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:44:53.926+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:44:53.924+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:44:53.926+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:44:53.947+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.530 seconds
[2025-08-22T18:45:24.638+0000] {processor.py:186} INFO - Started process (PID=234) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:45:24.641+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:45:24.644+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:45:24.644+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:45:25.135+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:45:25.133+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:45:25.135+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:45:25.157+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.526 seconds
[2025-08-22T18:45:55.853+0000] {processor.py:186} INFO - Started process (PID=247) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:45:55.854+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:45:55.857+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:45:55.857+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:45:56.342+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:45:56.340+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:45:56.343+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:45:56.364+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-22T18:46:27.073+0000] {processor.py:186} INFO - Started process (PID=260) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:46:27.075+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:46:27.078+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:46:27.077+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:46:27.565+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:46:27.564+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:46:27.566+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:46:27.587+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-22T18:46:58.285+0000] {processor.py:186} INFO - Started process (PID=273) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:46:58.286+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:46:58.289+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:46:58.289+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:46:58.778+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:46:58.776+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:46:58.778+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:46:58.799+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.522 seconds
[2025-08-22T18:47:29.588+0000] {processor.py:186} INFO - Started process (PID=286) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:47:29.590+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:47:29.593+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:47:29.592+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:47:30.077+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:47:30.075+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:47:30.078+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:47:30.101+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-22T18:48:00.795+0000] {processor.py:186} INFO - Started process (PID=299) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:48:00.797+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:48:00.800+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:48:00.799+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:48:01.281+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:48:01.280+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:48:01.282+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:48:01.304+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
[2025-08-22T18:48:32.006+0000] {processor.py:186} INFO - Started process (PID=312) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:48:32.009+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:48:32.012+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:48:32.011+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:48:32.495+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:48:32.493+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:48:32.496+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:48:32.519+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-22T18:49:03.226+0000] {processor.py:186} INFO - Started process (PID=325) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:49:03.227+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:49:03.231+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:49:03.230+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:49:03.751+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:49:03.749+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:49:03.751+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:49:03.772+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.553 seconds
[2025-08-22T18:49:34.551+0000] {processor.py:186} INFO - Started process (PID=338) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:49:34.553+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:49:34.556+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:49:34.556+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:49:35.040+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:49:35.038+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:49:35.041+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:49:35.063+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-22T18:50:05.751+0000] {processor.py:186} INFO - Started process (PID=351) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:50:05.752+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:50:05.755+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:50:05.754+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:50:06.241+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:50:06.240+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:50:06.242+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:50:06.262+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-22T18:50:36.989+0000] {processor.py:186} INFO - Started process (PID=364) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:50:36.991+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:50:36.994+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:50:36.994+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:50:37.475+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:50:37.473+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:50:37.475+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:50:37.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.515 seconds
[2025-08-22T18:51:08.186+0000] {processor.py:186} INFO - Started process (PID=377) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:51:08.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:51:08.190+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:51:08.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:51:08.675+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:51:08.674+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:51:08.676+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:51:08.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-22T18:51:39.389+0000] {processor.py:186} INFO - Started process (PID=390) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:51:39.391+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:51:39.394+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:51:39.393+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:51:39.875+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:51:39.873+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:51:39.875+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:51:39.900+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-22T18:52:10.689+0000] {processor.py:186} INFO - Started process (PID=403) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:52:10.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:52:10.694+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:52:10.693+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:52:11.175+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:52:11.174+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:52:11.176+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:52:11.196+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-22T18:52:41.883+0000] {processor.py:186} INFO - Started process (PID=416) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:52:41.884+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:52:41.887+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:52:41.887+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:52:42.374+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:52:42.372+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:52:42.374+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:52:42.396+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.521 seconds
[2025-08-22T18:53:13.083+0000] {processor.py:186} INFO - Started process (PID=435) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:53:13.085+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:53:13.088+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:53:13.087+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:53:13.566+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:53:13.564+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:53:13.566+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:53:13.586+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.510 seconds
[2025-08-22T18:53:44.344+0000] {processor.py:186} INFO - Started process (PID=449) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:53:44.345+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:53:44.349+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:53:44.349+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:53:44.831+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:53:44.829+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:53:44.832+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:53:44.855+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-22T18:54:15.644+0000] {processor.py:186} INFO - Started process (PID=462) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:54:15.645+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:54:15.648+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:54:15.647+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:54:16.133+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:54:16.132+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:54:16.134+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:54:16.154+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.518 seconds
[2025-08-22T18:54:46.871+0000] {processor.py:186} INFO - Started process (PID=475) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:54:46.873+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:54:46.876+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:54:46.875+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:54:47.356+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:54:47.355+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:54:47.357+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:54:47.377+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.513 seconds
[2025-08-22T18:55:18.093+0000] {processor.py:186} INFO - Started process (PID=488) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:55:18.094+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:55:18.097+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:55:18.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:55:18.590+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:55:18.588+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:55:18.590+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:55:18.609+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.524 seconds
[2025-08-22T18:55:49.315+0000] {processor.py:186} INFO - Started process (PID=501) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:55:49.317+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:55:49.320+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:55:49.319+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:55:49.806+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:55:49.804+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:55:49.806+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:55:49.828+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.520 seconds
[2025-08-22T18:56:20.538+0000] {processor.py:186} INFO - Started process (PID=514) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:56:20.539+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:56:20.542+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:56:20.542+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:56:21.024+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:56:21.023+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:56:21.025+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:56:21.044+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-22T18:56:51.751+0000] {processor.py:186} INFO - Started process (PID=527) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:56:51.753+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:56:51.756+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:56:51.755+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:56:52.240+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:56:52.238+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:56:52.240+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:56:52.260+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.517 seconds
[2025-08-22T18:57:23.014+0000] {processor.py:186} INFO - Started process (PID=540) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:57:23.015+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:57:23.019+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:57:23.019+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:57:23.508+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:57:23.506+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:57:23.508+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:57:23.528+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.523 seconds
[2025-08-22T18:57:54.210+0000] {processor.py:186} INFO - Started process (PID=553) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:57:54.212+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:57:54.215+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:57:54.214+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:57:54.697+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:57:54.695+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:57:54.697+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:57:54.717+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.514 seconds
[2025-08-22T18:58:25.433+0000] {processor.py:186} INFO - Started process (PID=566) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:58:25.435+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:58:25.438+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:58:25.438+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:58:25.934+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:58:25.933+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:58:25.935+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:58:25.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.530 seconds
[2025-08-22T18:58:56.660+0000] {processor.py:186} INFO - Started process (PID=579) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:58:56.662+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:58:56.666+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:58:56.665+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:58:57.152+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:58:57.150+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:58:57.152+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:58:57.172+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.519 seconds
[2025-08-22T18:59:27.874+0000] {processor.py:186} INFO - Started process (PID=592) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:59:27.876+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:59:27.879+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:59:27.879+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:59:28.356+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:59:28.355+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:59:28.357+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:59:28.378+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.510 seconds
[2025-08-22T18:59:59.088+0000] {processor.py:186} INFO - Started process (PID=605) to work on /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:59:59.089+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/bank_transactions.py for tasks to queue
[2025-08-22T18:59:59.093+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:59:59.092+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:59:59.587+0000] {logging_mixin.py:190} INFO - [2025-08-22T18:59:59.585+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/bank_transactions.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/bank_transactions.py", line 14, in <module>
    from lib.clickhouse_operator_extended import ClickHouseOperatorExtended
ModuleNotFoundError: No module named 'lib'
[2025-08-22T18:59:59.587+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/bank_transactions.py
[2025-08-22T18:59:59.610+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/bank_transactions.py took 0.529 seconds
